k8s/agones/chart/agones-mp/templates/application.yaml

```diff
@@ -0,0 +1,65 @@
+    marketplace.cloud.google.com/deploy-info: '{"partner_id": "click-to-deploy-containers", "product_id": "agones", "partner_name": "Google Click to Deploy"}'
+spec:
+  descriptor:
+    type: "Agones"
+    versions: '0.8.1'
+
+    description: |-
+      Agones is a library for hosting, running and scaling dedicated game servers on Kubernetes.
+
+      # Support
+      Google does not offer support for this solution. However, community support is available on the Agones
+      [mailing list](https://groups.google.com/forum/#!forum/agones-discuss) or on
+      [Slack](https://join.slack.com/t/agones/shared_invite/enQtMzE5NTE0NzkyOTk1LWQ2ZmY1Mjc4ZDQ4NDJhOGYxYTY2NTY0NjUwNjliYzVhMWFjYjMxM2RlMjg3NGU0M2E0YTYzNDIxNDMyZGNjMjU).
+
+    maintainers:
+    - name: Google Click to Deploy
+      url: https://cloud.google.com/solutions/#click-to-deploy
+
+    links:
+    - description: 'Agones Homepage'
+      url: https://agones.dev/
+    - description: 'Agones Documentation'
+      url: https://agones.dev/site/docs/
+    - description: 'Getting Started'
+      url: https://agones.dev/site/docs/getting-started/
+
+    notes: |-
+      Now that you have deployed Agones into your cluster, please follow [these instructions](https://agones.dev/site/docs/getting-started/) to get started with Agones.
+      Please note that you should create your resources inside the namespace that was selected during the installation for click-to-deploy Agones to work.
```

Comment:
```suggestion
      Please note that you should create your resources inside the namespace that was selected during the installation for Click to Deploy Agones to work.
```

Reply:
Rephrased it with the documentation update, and that section was dropped.